### ISSUE TITLE
[Bug] Hub & Crimson Isle - Fancy Warp Menu

### ISSUE THREAD

**Issue by CubedTemp** [Fix3dll/SkyblockAddons#17]
### Bug Description

Hub in Fancy Warp Menu when hovered over is zoomed under instead of over the other elements
![image](https://github.com/Fix3dll/SkyblockAddons/assets/89503528/ecd58376-849d-456f-8e79-5d71c5db4375)

Crimson Isle and Hub hovered leave the picture of the island below the zoomed in version

No hover:
![image](https://github.com/Fix3dll/SkyblockAddons/assets/89503528/7814fdbb-b1ba-4d86-bc8d-b951b6fef324)
![image](https://github.com/Fix3dll/SkyblockAddons/assets/89503528/e6961d32-36cc-4291-b293-6bec10528f18)

Hovered:
![image](https://github.com/Fix3dll/SkyblockAddons/assets/89503528/3dd6d968-ef93-47d9-add2-8c3482bb02cd)
![image](https://github.com/Fix3dll/SkyblockAddons/assets/89503528/63ed1302-e98d-4d09-b0fe-146bba36155e)

Notice the 4 towers grouped together on the hub (hovered)

### SBA Version

1.8.5+150

### Other Installed Mods

neustats output:
```md
# System Stats
[OS][Windows 11]
[CPU][24x 13th Gen Intel(R) Core(TM) i7-13700K]
[Display][1920x1009 (NVIDIA Corporation)]
[GPU][NVIDIA GeForce RTX 3070/PCIe/SSE2]
[GPU Driver][4.6.0 NVIDIA 546.01]
[Maximum Memory][48939MB]
[Shaders][TRUE]
# Java Stats
[Java][1.8.0_371 64bit]
[Memory][ 77% 1409/1820MB]
[Memory Allocated][ 98% 1789MB]
# Game Stats
[FPS][144]
[Loaded Mods][57]
[Forge][11.15.1.2318]
[Optifine][TRUE]
# Neu Settings
[On SkyBlock][TRUE]
[Mod Version][2.1.1+Alpha-24.49fcd68]
[SB Profile][Mango]
[Has Advanced Tab][TRUE]
# Repo Stats
[Last Commit][1e3b392909c4a2ae14f35537b9ced0cdd9fdf41b]
[Repo Location][https://github.com/NotEnoughUpdates/NotEnoughUpdates-REPO/tree/prerelease]
[Using Backup][false]
[Loaded Items][6036]
[Lowest Bin API Location][moulberry.codes]
```

### Operating System

Windows 11

**Comment by Fix3dll** [comment]
@CubedTemp could you try with latest build? Thanks for your reporting.

**Comment by CubedTemp** [comment]
Fixed as of SBA 1.8.5+152.
